package.json

```diff
@@ -41,7 +41,8 @@
     "uglify"
   ],
   "engines": {
-    "node": "^14.13.1 || >=16.0.0"
+    "node": ">=14.13 || >=16 || >=20",
+    "npm": ">=9"
```

Comment:
This line shouldn't be there in my opinion.

This could lead to some issues where yarn or something else is used on the consumer side and the package manager applies this for the whole setup step.

The npm version is only relevant for CI, not for consumers.